code/__HELPERS/unsorted.dm

```diff
@@ -760,7 +760,10 @@ proc/anim(turf/location as turf,target as mob|obj,a_icon,a_icon_state as text,fl
 	var/image/progbar
 	for (var/i = 1 to numticks)
 		if(user.client)
-			progbar = make_progress_bar(i, numticks, target)
+			var/bar_target = target

```

Comment:
I didn't market this as a bugfix, goofball - I fixed this bug because it interfered with one of the changes I was making - passing in `null` as a target results in it runtiming due to make_progress_bar expecting something to put the bar over, and without the fix it'll choke.